set.mm

```diff
@@ -148416,6 +149653,19 @@ Splicing words (substring replacement)
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
 $)
 
+  $c reverse $.
+
+  $( Syntax for the word reverse operator. $)
+  creverse $a class reverse $.
+
+  ${
+    $d s x $.
+    $( Define an operation which reverses the order of symbols in a word.
+       (Contributed by Stefan O'Rear, 26-Aug-2015.) $)
```

Comment:
Maybe add some synonyms, e.g., `This operation is also known as "word reversal" and "word mirroring".` This way, it will show up when someone searches for these expressions.

Reply:
Done.